packages/evolution-common/src/services/interviews/__tests__/interview.test.ts

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2023, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+import { InterviewResponses, SectionActionStatus } from '../interview';
+import { Household } from '../../interviewObjects/Household';
+import { Place } from '../../interviewObjects/Place';
+import { Vehicle } from '../../interviewObjects/Vehicle';
+import { Person } from '../../interviewObjects/Person';
+
+type CustomSurvey = { customSurveyAttribute: string };
+type CustomHousehold = { customHouseholdAttribute: string };
+type CustomPerson = { customPersonAttribute: string };
+type CustomPlace = { customPlaceAttribute: string };
+type CustomVehicle = { customVehicleAttribute: string };
+type CustomVisitedPlace = { customVisitedPlaceAttribute: string };
+type CustomTrip = { customTripAttribute: string };
+type CustomSegment = { customSegmentAttribute: string };
+
+describe('InterviewResponses type tests', () => {
+    test('Valid InterviewResponses values', () => {
+        const interviewResponses: InterviewResponses<CustomSurvey, CustomHousehold, CustomPerson, CustomPlace, CustomVehicle, CustomVisitedPlace, CustomTrip, CustomSegment> = {
```

Comment:
Most of the added tests are trivial, but it does make sure that this object compiles. Maybe add a comment explaining that this tests more the compilation of this object than behavior. And everywhere such tests exist